ceph-releases/ALL/opensuse__leap/__DOCKERFILE_INSTALL__

```diff
@@ -0,0 +1,13 @@
+( echo 'zypper add repos' && \
```

Comment:
A lot of these files seem like they are the same as the `42.3` files. Probably the better solution is to replace the existing `opensuse` directory with `opensuse__leap` so that 42.3 and 15.0 can exist in the same directory structure without duplicating code. Then the 42.3 ceph-container image will use the `opensuse/leap:42.3` image.

Reply:
fixed.